bin/client/main.rs

```diff
@@ -108,6 +109,50 @@ async fn main() -> std::result::Result<(), ClientError> {
             .await?;
             println!("{}", toml::to_string_pretty(&label)?);
         }
+        SubCommand::CreateKey(create_opts) => {
+            let dir = create_opts.secret_file.unwrap_or_else(|| {
+                default_config_dir()
+                    .unwrap_or_else(|| PathBuf::from("."))
+                    .join("secret_keys.toml")
+            });
+            println!("Writing keys to {}", dir.to_string_lossy());
```

Comment:
I think `dir.display()` is preferred here rather than doing a string conversion. Same goes on all other similar lines